pages/book/create.js

```diff
@@ -0,0 +1,52 @@
+import Link from "next/link";
+import styled from "styled-components";
+import BookForm from "@/components/BookForm/BookForm";
+import Bottom from "@/components/Bottom/Bottom";
+
+export default function CreateBookPage({ onNewSubmit }) {
+  return (
+    <StyledMain>
+      <BookForm onNewSubmit={onNewSubmit} />
+      <Bottom>
+        <StyledLinkPlaceholder />
+        <StyledLink href={"/"}>Home</StyledLink>
```

Comment:
you don't need the brackets here

```suggestion
        <StyledLink href="/">Home</StyledLink>
```

Reply:
Remove curly brackets. Thats why VSCode suggest it.